app/src/main/java/pl/futurecollars/invoicing/service/IdService.java

```diff
@@ -0,0 +1,39 @@
+package pl.futurecollars.invoicing.service;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+
+public class IdService {
+
+    private final Path idFilePath;
+    private final FileService fileService;
+
+    private int nextId = 1;
+
+    public IdService(Path idFilePath, FileService fileService) {
+        this.idFilePath = idFilePath;
+        this.fileService = fileService;
+
+        try {
```

Comment:
Try to move creating a file to FileService and make it generic

Reply:
I try to do it as we discussed yesterday. My changes did make several issues in the testing area, I got most of it covered but with the 4 remaining, I got stuck. Could you please aid me with it?

Reply:
nvm it. Got it done